Marlin/src/pins/stm32f4/pins_Fly_RRF_E3_V1.h

```diff
@@ -0,0 +1,342 @@
+  #endif
+
+  // Alter timing for graphical display
+  #if ENABLED(LCD_FOR_MELZI)                      // LCD_FOR_MELZI default timing is too fast. This works but may be reduced.
+    #define BOARD_ST7920_DELAY_1             200
+    #define BOARD_ST7920_DELAY_2             400
+    #define BOARD_ST7920_DELAY_3            1200
+  #elif IS_U8GLIB_ST7920
+    #define BOARD_ST7920_DELAY_1              96
+    #define BOARD_ST7920_DELAY_2              48
+    #define BOARD_ST7920_DELAY_3             600
+  #endif
+
+#endif // HAS_WIRED_LCD
+
+#if ALL(TOUCH_UI_FTDI_EVE, LCD_FYSETC_TFT81050)
+
+  #ifndef NO_CONTROLLER_CUSTOM_WIRING_WARNING
+    #error "CAUTION! LCD_FYSETC_TFT81050 requires wiring modifications. See 'pins_Fly_RRF_E3_V1.h' for details. (Define NO_CONTROLLER_CUSTOM_WIRING_WARNING to suppress this warning.)"
+  #endif
+
+  /** FYSETC TFT TFT81050 display pinout
+   *
+   *                    Board                          Display
+   *                   ------                          ------
+   * (SD_DET)    PE12 | 1  2 | PE11 (BEEPER)       5V |10  9 | GND
+   * (MOD_RESET) PE10 | 3  4 | RESET            RESET | 8  7 | (SD_DET)
+   * (SD_CS)     PE9    5  6 | PE8            (MOSI)  | 6  5 | (LCD_CS)
+   * (LCD_CS)    PE7  | 7  8 | PB1            (SD_CS) | 4  3 | (MOD_RESET)
+   *             GND  | 9 10 | 5V             (SCK)   | 2  1 | (MISO)
+   *                   ------                          ------
+   *                   EXP1                            EXP1
+   *
+   *
+   * Needs custom cable:
+   *
+   *    Board   Adapter   Display
+   *   ----------------------------------
+   *   EXP1-10 ---------- EXP1-10  5V
+   *   EXP1-9 ----------- EXP1-9   GND
+   *   SPI1-4 ----------- EXP1-6   MOSI
+   *   EXP1-7 ----------- EXP1-5   LCD_CS
+   *   SPI1-3 ----------- EXP1-2   SCK
+   *   EXP1-5 ----------- EXP1-4   SD_CS
+   *   EXP1-4 ----------- EXP1-8   RESET
+   *   EXP1-3 ----------- EXP1-3   MOD_RST
+   *   SPI1-1 ----------- EXP1-1   MISO
+   *   EXP1-1 ----------- EXP1-7   SD_DET
+   */
+
+  #define CLCD_SPI_BUS                         1  // SPI1 connector
+
+  #define BEEPER_PIN                 EXP1_02_PIN
+
+  #define CLCD_MOD_RESET             EXP1_03_PIN
+  #define CLCD_SPI_CS                EXP1_07_PIN
+
+#endif // TOUCH_UI_FTDI_EVE && LCD_FYSETC_TFT81050
+
+//
+// SD Support
+//
+
+#ifndef SDCARD_CONNECTION
+  #define SDCARD_CONNECTION              ONBOARD
+#endif
+
+#if SD_CONNECTION_IS(ONBOARD)
+  #define ONBOARD_SDIO                            // Use SDIO for onboard SD
+  //#define SDIO_CLOCK                  48000000
+  #define SD_DETECT_PIN                     PA15
+#elif SD_CONNECTION_IS(CUSTOM_CABLE)
+  #error "SD CUSTOM_CABLE is not compatible with Fly RRF E3 V1."
+#endif
+
+// TODO ? board has inbuilt WiFi (ESP8266) but probably already runs a version of DuetWebServer if you've ever used it on RRF before and i have no idea what you'd need to flash it with to get it to work
```

Comment:
Review how this is done on other boards like the BTT E3 RRF: https://github.com/MarlinFirmware/Marlin/blob/f0b115a9993c0bf6eace56ad2b8c2d7944dfdd8a/Marlin/src/pins/stm32f4/pins_BTT_E3_RRF.h#L397-L406